.github/workflows/snyk.yml

```diff
@@ -14,18 +14,20 @@ jobs:
   # we do not scan images here, they're scanned here: https://app.snyk.io/org/argoproj/projects
 
   golang:
+    name: Scan Go deps
     if: github.repository == 'argoproj/argo-workflows'
     runs-on: ubuntu-latest
     env:
       SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
     steps:
       - uses: actions/checkout@v3
-      - name: Run Snyk to check for vulnerabilities
+      - name: Run Snyk to check for Go vulnerabilities
         uses: snyk/actions/golang@master
         with:
           args: --severity-threshold=high
 
   node:
+    name: Scan Node deps
```

Comment:
Not necessary. It's already under a `node` section

Reply:
It makes more sense in context; the below screenshot just says "Snyk / node":

![Screenshot_2023-09-26-10-36-32-04_40deb401b9ffe8e1df2f1cc5ba480b12](https://github.com/argoproj/argo-workflows/assets/4970083/7e93f43f-1b6f-415a-97b7-b1ffc76fc65e)

That is not entirely clear what it does since Snyk has many functionalities (see also automated PRs #11844, scanning Docker images, etc). This makes it very explicit at first glance